data/json/items/gunmod/brass_catcher.json

```diff
@@ -17,5 +17,24 @@
     "pocket_data": [ { "pocket_type": "CONTAINER", "rigid": true, "max_contains_volume": "249 ml", "max_contains_weight": "1 kg" } ],
     "flags": [ "BRASS_CATCHER" ],
     "min_skills": [ [ "weapon", 1 ] ]
+  },
+  {
+    "id": "big_brass_catcher",
```

Comment:
This seems excessively large, do you have any examples of brass catchers on the market that are this big?

Reply:
https://www.cabelas.ca/product/113208/caldwell-universal-brass-catcher-sa Fits 100 .223 casings which if the item def for the 223 casing is accurate, is just about bang on. But also, aren't the brass catchers we have supposed to make crude makeshift ones? I don't see why someone couldn't make one, but bigger

Reply:
Considering the number of zeds/mutants that the CDDA survivors are dealing with on a regular basis, it makes sense that they'd need catchers that can hold more rounds than modern folks might, and it's not a far stretch to imagine you could just strip the bag off of one and replace it with a bigger bag. :P
In that case it would be a recipe, but I've already fought enough with those for another batch of items that might see PR some time soon. If these two do get accepted, there's a small change I'd need to put into PR as well to add the offset grip to the other underbarrel grips' blacklist as well (I have done this already on my PC but haven't done a branch for it yet) and could add the recipe for the big brass catcher at the same time as a sort of 'part 2' to this.